packages/base/src/formbuilder/objectform/baseform.tsx

```diff
@@ -173,25 +173,66 @@ export class BaseForm extends React.Component<IBaseFormProps, IBaseFormStates> {
       if (k === 'opacity') {
         uiSchema[k] = {
           'ui:field': (props: any) => {
-            const handleChange = (event: CustomEvent) => {
+            const [inputValue, setInputValue] = React.useState(
+              props.formData.toFixed(1)
+            );
+
+            React.useEffect(() => {
+              setInputValue(props.formData.toFixed(1));
+            }, [props.formData]);
+
+            const handleSliderChange = (event: CustomEvent) => {
               const target = event.target as any;
               if (target && '_value' in target) {
-                const value = parseFloat(target._value);
-                props.onChange(value);
+                const sliderValue = parseFloat(target._value); // Slider value is in 0–10 range
+                const normalizedValue = sliderValue / 10; // Normalize to 0.1–1 range
```

Comment:
I'm confused by the 0.1 choice here? Why not the 0-1 range?

Reply:
Because opacity need to be a multiple of `0.1`, as defined in layer schemas - https://github.com/geojupyter/jupytergis/blob/main/packages/schema/src/schema/imageLayer.json#L16

Reply:
Fundamentally we might want to avoid making a layer fully transparent

Reply:
I'm not sure I agree with this, but no hard feeling. We can discuss that in an issue after merging this PR :)